tools/build_file_generator/third_party_deps/third_party_deps_updater.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
```

Comment:
How did we do this before this script?

Reply:
In the beginning I used `bazel query 'attr("visibility", "//visibility:public", //third_party/...)'` command and fill in deps in the BUILD file manually. 
We didn't have automation way for updating the BUILD file after adding/removing a third-party dependency in the project.